src/App.jsx

```diff
@@ -19,7 +19,7 @@ const PATHNAME = (new URL(document.location)).pathname.replace(/\/+$/, '');
 const API_GW_URL = 'https://api.zeer0.com/v001';
 const EMAIL_CONTENT_URL = `${API_GW_URL}/moogle/email`;
 const EMAILS_LIST_URL = `${API_GW_URL}/moogle/email/list`;
-const DEFAULT_FQDN = HOST;
+const DEFAULT_FQDN = HOST.startsWith('localhost') ? 'moogle.cc' : HOST;
```

Comment:
This needs to fixed. Can't have hackers guessing that they can run this code on localhost with any domain.